railgun/test/integration/suite_test.go

```diff
@@ -138,6 +145,13 @@ func deployControllers(ctx context.Context, ready chan ktfkind.ProxyReadinessEve
 			return err
 		}
 	}
+	// ensure the alternative namespace is created
+	elsewhereNS := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "elsewhere"}}
+	if _, err := cluster.Client().CoreV1().Namespaces().Create(ctx, elsewhereNS, metav1.CreateOptions{}); err != nil {
+		if !errors.IsAlreadyExists(err) {
+			return err
+		}
+	}
```

Comment:
I have the same questions here as I did above, why is this a glob?

Reply:
No good reason. Wrote this bit before adding the constant and never fixed it, thanks for the catch.